backend/src/main/java/ca/bc/gov/app/exception/UnableToProcessRequestException.java

```diff
@@ -7,11 +7,15 @@
 
 @ResponseStatus(HttpStatus.UNPROCESSABLE_ENTITY)
 public class UnableToProcessRequestException extends ResponseStatusException {
+
+  private static final long serialVersionUID = -4004447722417852411L;
```

Comment:
You don't need to add the serial version anymore. This is only used for binary serialization and serves no purpose when handling JSON data

Reply:
Thanks. My IDE was complaining for that, so I added it :-) The error was: "The serializable class ValidationError does not declare a static final serialVersionUID field of type long"